engine/release-notes/index.md

```diff
@@ -22,6 +22,63 @@ for Docker Engine.
+* Prevent sending `SIGURG` signals to container on Linux and macOS. The Go runtime
+  (starting with Go 1.14) uses `SIGURG` signals internally as an interrupt to
+  support preemptable syscalls. In situations where the Docker CLI was attached
+  to a container, these interrupts were forwarded to the container. This fix
+  changes the Docker CLI to ignore `SIGURG` signals [docker/cli#3107](https://github.com/docker/cli/pull/3107),
+  [moby/moby#42421](https://github.com/moby/moby/pull/42421).
+
+### Builder
+
+* Update BuildKit to version v0.8.3-3-g244e8cde [moby/moby#42448](https://github.com/moby/moby/pull/42448):
+    * Transform relative mountpoints for exec mounts in the executor to work around
+      a breaking change in runc v1.0.0-rc94 and up. [moby/buildkit#2137](https://github.com/moby/buildkit/pull/2137).
+    * Add retry on image push 5xx errors. [moby/buildkit#2043](https://github.com/moby/buildkit/pull/2043).
+    * Fix build-cache not being invalidated when renaming a file that is copied using
+      a `COPY` command with a wildcard. Note that this change invalidates
+      existing build caches for copy commands that use a wildcard. [moby/buildkit#2018](https://github.com/moby/buildkit/pull/2018).
+    * Fix build-cache not being invalidated when using mounts [moby/buildkit#2076](https://github.com/moby/buildkit/pull/2076).
+* Fix build failures when `FROM` image is not cached when using legacy schema 1 images [moby/moby#42382](https://github.com/moby/moby/pull/42382).
+
+### Logging
+
+* Update the hcsshim SDK to make daemon logs on Windows less verbose [moby/moby#42292](https://github.com/moby/moby/pull/42292).
+
+### Rootless
+
+* Fix capabilities not being honoured when an image was built on a daemon with
+  user-namespaces enabled [moby/moby#42352](https://github.com/moby/moby/pull/42352).
+
+### Networking
+
+* Update libnetwork to fix publishing ports on environments with kernel boot
+  parameter `ipv6.disable=1`, and to fix a deadlock causing internal DNS lookups
+  to fail [moby/moby#42413](https://github.com/moby/moby/pull/42413).
+
+### Contrib
+
+* Update rootlesskit to v0.14.2 to fix a timeout when starting the userland proxy
+  with the `slirp4netns` port driver [moby/moby#42294](https://github.com/moby/moby/pull/42294).
+* Fix "Device or resource busy" errors when running docker-in-docker on a rootless
+  daemon [moby/moby#42342](https://github.com/moby/moby/pull/42342).
+
+### Packaging
+
+* Update containerd to v1.4.6, runc v1.0.0-rc95 to address [CVE-2021-30465](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-30465)
+  [moby/moby#42398](https://github.com/moby/moby/pull/42398), [moby/moby#42395](https://github.com/moby/moby/pull/42395),
+  [ocker/containerd-packaging#234](https://github.com/docker/containerd-packaging/pull/234)
+* Update containerd to v1.4.5, runc v1.0.0-rc94 [moby/moby#42372](https://github.com/moby/moby/pull/42372),
```

Comment:
Do we want to include this entry as well?  containerd 1.4.5 is replaced by 1.4.6 and runc 1.0.0-rc94 is replaced by 1.0.0-rc95.

Reply:
I was a bit in doubt; reason I included them both was that the v1.4.5 update (and runc 1.0.0-rc94) has a much larger changelog; I thought mentioning it (and providing links to the pull requests) could make those changes more discoverable.

Reply:
We could write this in one entry with all the PRs?  Or we could leave it as two separate entries.  Either way seems fine to me.